client/extensions/woocommerce/components/woocommerce-colophon/index.js

```diff
@@ -13,13 +14,13 @@ import WooCommerceLogo from '../woocommerce-logo';
 const WooCommerceColophon = ( { translate } ) => {
 	return (
 		<div className="woocommerce-colophon">
-			<span>
+			<ExternalLink icon={ false } target="_blank" href="https://woocommerce.com">
```

Comment:
Should we set the icon to `false`? cc @davidakennedy and @enriquesanchez for an accessibility perspective.

Reply:
Does this link have to be opened on a new tab/window? In most situations, you wouldn't want to open new tabs/windows because they can be disorienting for some, especially those who have difficulty perceiving visual content.

If it really has to be opened on a new tab/window, then it's recommended to give the user some warning.

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/33845/commits/1dc02af16fec38297f6194e9049ab6e0626aae14